packages/harmony/src/components/comments/IconText/IconText.tsx

```diff
@@ -0,0 +1,30 @@
+import { Flex } from 'components/layout'
+import { Text } from 'components/text'
+
+import { IconTextProps } from './types'
+
+export const IconText = ({
+  icons = [],
+  text,
+  color = 'default'
+}: IconTextProps) => {
+  const separator = (
+    <Text variant='body' color='default' css={{ fontSize: 8 }}>
+      •
+    </Text>
+  )
+
+  return (
+    <Flex h={16} gap='xs' alignItems='center'>
```

Comment:
H=l

Reply:
wat?

Reply:
p sure he means this lol
```suggestion
    <Flex h='l gap='xs' alignItems='center'>
```

Reply:
ooh i thought it was an I not an l. (uppercase i not lowercase L)